src/data/store/selectors/linkSaturation.selectors.ts

```diff
@@ -4,8 +4,12 @@
 
 import { RootState } from '../createStore';
 
-export const getTotalOpsForGraph = (temporalEpoch: number) => (state: RootState) =>
-    state.linkSaturation.linksPerTemporalEpoch[temporalEpoch]?.totalOps || 0;
+export const getTotalOps = (temporalEpoch: number, chipId?: number) => (state: RootState) =>
```

Comment:
i also dont like relying on chipId being undefined. we should have two separate selectors to query totalOps for chip or totalOps for temporalEpoch with the selctor name reflacting that.

Reply:
When we load a temporal epoch, `chipId` is undefined, we follow this pattern for a while now... 👀

Do you want me to change it everywhere?